### PR TITLE
feat: add new roadmap page

### PR DESCRIPTION
**Pull request by melaniebmn**
# Summary | Résumé

Add a new roadmap page.

### CODE REVIEW

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Review by daine — APPROVED** [review]

File: src/en/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  hideMain: true
+---
+
+# Roadmap
+## Timeline: Q1 (Apr - Jun 2024) 
+
+### Theme: Product Growth
+
+#### Items:
+- **Support for Server Side Rendering frameworks**
```
Comment (daine) [inline src/en/roadmap.md:19]: @adorayi oh just noticed, should be mention that it's currently specific to React?
```suggestion
- **Support for React with Server Side Rendering frameworks**
```

**Comment by adorayi** [comment]
> LGTM! @adorayi are we linking to this page from somewhere else in the site?

No need to link it for now, we can share the direct url for now and then consider the full site structure another time.

File: src/en/roadmap.md (hunk 1 above)
Comment (adorayi) [reply src/en/roadmap.md:19]: That was more of a general statement which the sub bullet expands on with details (NextJs). I don't think we need to change this now, we can revisit later. Thanks.

**Pull request merged**